packages/ibm-products/src/components/ProductiveCard/ProductiveCard.docs-page.js

```diff
@@ -43,6 +43,8 @@ const DocsPage = () => (
       },
       {
         story: stories.WithOverflow,
+        description:
+          'To include an overflow menu within the productive card, you should provide an array of objects. These will map to the overflow items, all supported props from `OverflowItem` will be passed through to the overflow menu items created via the `overflowActions` prop.',
```

Comment:
Think this should be a new sentence? Or colon?

```suggestion
          'To include an overflow menu within the productive card, you should provide an array of objects. These will map to the overflow items. All supported props from `OverflowItem` will be passed through to the overflow menu items created via the `overflowActions` prop.',
```

```suggestion
          'To include an overflow menu within the productive card, you should provide an array of objects. These will map to the overflow items; all supported props from `OverflowItem` will be passed through to the overflow menu items created via the `overflowActions` prop.',
```